src/FixedDataTableCell.js

```diff
@@ -98,16 +98,23 @@ class FixedDataTableCell extends React.Component {
     reorderingDisplacement: 0
   }
 
+  // TODO (pradeep): Remove this when merging to Beta (acts as a quick check to see if cells are mounted)
+  componentDidMount() {
+    console.log("cell mounted");
```

Comment:
Found that cell mounts only happen during initial scrolling, due to the behavior explained [here](https://github.com/schrodinger/fixed-data-table-2/pull/430/files#r275101300). 
(@wcjordan , we shouldn't forget to investigate possible approaches in later iterations.)

Otherwise not a single cell seems to mounted.